www/index.js

```diff
@@ -227,8 +229,10 @@ HotPush.prototype._loadLocalVersion = function(callback) {
 * Callback for async call to version files
 */
 HotPush.prototype._verifyVersions = function() {
```

Comment:
changes are done!
I've also bumped the contentsync dependecy to the newest version.
please check it out ;-)